dpdata/lammps/dump.py

```diff
@@ -83,7 +83,7 @@ def safe_get_posi(lines,cell,orig=np.zeros(3)) :
     if sf:
         posis = (posis%1.)@cell # convert xsu to xs first
     else:
-        posis = (posis - orig)%np.linalg.norm(cell,axis=1)
+        posis = (posis - orig)%cell.sum(axis=0)
```

Comment:
The 0th dim of cell is the frame. `cell.sum(axis=0)` sums the cell tensors of all frames. It doesn't make sense to me.

Reply:
The safe_get_posi function handles one frame each time. When the safe_get_posi function is called from system_data, the cell argument passed to safe_get_posi is returned from dumpbox2box function, which is always a 3*3 array.

Reply:
I see that cell is of shape [3,3], but`%cell.sum(axis=0)` would not give the right wrapped coordinated in a triclinic cell. 

The correct way would be:
1. convert the physical coordinates to scaled coordinates. Notice that scale cell is a cubic
2. wrap the scaled coordinate by `posis%1.`
3. convert the scaled coordinates back to the physical coordinates.

Reply:
It seems that [x,y,z]@inv(cell) can accurately scale the physical coordinates, where inv is the inverse operation.